tests/nnstreamer_sink/unittest_sink.cc

```diff
@@ -35,6 +35,8 @@
     goto error; \
   }
 
+#define TEST_TIME_OUT_MSEC 2000
```

Comment:
Instead of #define, how about use const?

Reply:
I will update it!